find_me_a_sandwich/features/step_definitions/favorites_loggedout_steps.rb

```diff
@@ -0,0 +1,7 @@
+When(/^I visit the favorite meals list$/) do
```

Comment:
This filename is a bit too specific.  Typically, steps are organized by model or section (e.g. `favorites_steps`, `session_steps`, etc.)